assets/src/components/v2/triptych/no_data.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ComponentType } from "react";
+import LcdNoData from "Components/v2/lcd/no_data";
+
+interface Props {
+  show_alternatives: boolean;
+}
+
+const NoData: ComponentType<Props> = ({
+  show_alternatives: showAlternatives,
+}) => {
+  // TODO: We likely want to show something different from the usual when we fail to fetch data for this screen type.
```

Comment:
(Do we have designs for this? I'm assuming we'd prefer to just show some baked-in PSA image or something instead of telling riders the app, which they weren't already keeping a particularly close eye out for since it's mixed in with ads, broke. Like an employee at a store rushing over to inform you that some specific item is out of stock when you weren't even looking for it.)

Reply:
Yeahhh no designs for this. I'd guess we'd fall back to evergreen widget (which will have the psas). Hopefully we wouldn't often run into a situation where even that widget couldn't render? 😬  So I don't think we'd want this frontend component.

Reply:
Yeah I'm talking about when we lose network connection. I guess if that happened then OFM would hopefully look into it, though.